docs/install-config/harbor-ha-helm.md

```diff
@@ -46,7 +46,9 @@ Configure the followings items in `values.yaml`, you can also set them as parame
 - **External Redis**
    Set the `redis.type` to `external` and fill the information in `redis.external` section.
 
-   As the Redis client used by Harbor's upstream projects doesn't support `Sentinel`, Harbor can only work with a single entry point Redis. You can refer to this [guide](https://community.pivotal.io/s/article/How-to-setup-HAProxy-and-Redis-Sentinel-for-automatic-failover-between-Redis-Master-and-Slave-servers) to setup a HAProxy before the Redis to expose a single entry point.
+   Harbor have supported redis `Sentinel` mode from 2.1.0, you can enable this by setting the `sentinel_master_set`, and setting the `host` as format `<host_sentinel1>:<port_sentinel1>,<host_sentinel2>:<port_sentinel2>,<host_sentinel3>:<port_sentinel3>`.
```

Comment:
Harbor introduced redis `Sentinel` mode support in 2.1.0. You can enable this by setting `sentinel_master_set` and `host` to `<host_sentinel1>:<port_sentinel1>,<host_sentinel2>:<port_sentinel2>,<host_sentinel3>:<port_sentinel3>`.

Reply:
fixed